pkg/apis/eventing/v1/broker_validation.go

```diff
@@ -67,22 +69,30 @@ func (b *Broker) CheckImmutableFields(ctx context.Context, original *Broker) *ap
 		return nil
 	}
 
-	// Make sure you can't change the class annotation.
-	diff, err := kmp.ShortDiff(original.GetAnnotations()[BrokerClassAnnotationKey], b.GetAnnotations()[BrokerClassAnnotationKey])
-
-	if err != nil {
+	// Only Subscriber and Reply are mutable.
```

Comment:
Copy-pasta from somewhere else I believe (?)

```suggestion
	// Only Delivery options are mutable.
```